pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ProducerBuilder.java

```diff
@@ -275,15 +275,16 @@
     /**
      * Set the compression type for the producer.
      *
-     * <p>By default, message payloads are not compressed. Supported compression types are:
+     * <p>By default, message payloads are not compressed.
+     * <p>Supported compression types are:
      * <ul>
-     * <li>{@link CompressionType#NONE}: No compression (Default)</li>
-     * <li>{@link CompressionType#LZ4}: Compress with LZ4 algorithm. Faster but lower compression than ZLib</li>
-     * <li>{@link CompressionType#ZLIB}: Standard ZLib compression</li>
-     * <li>{@link CompressionType#ZSTD} Compress with Zstandard codec. Since Pulsar 2.3. Zstd cannot be used if consumer
-     * applications are not in version >= 2.3 as well</li>
-     * <li>{@link CompressionType#SNAPPY} Compress with Snappy codec. Since Pulsar 2.4. Snappy cannot be used if
-     * consumer applications are not in version >= 2.4 as well</li>
+     * <li>{@link CompressionType#NONE}: No compression (default)</li>
+     * <li>{@link CompressionType#LZ4}: Compress with LZ4 algorithm. Faster but lower compression than ZLib.</li>
+     * <li>{@link CompressionType#ZLIB}: Standard ZLib compression.</li>
+     * <li>{@link CompressionType#ZSTD} Compress with Zstandard codec. Since Pulsar 2.3, Zstandard can only be used
```

Comment:
```suggestion
     * <li>{@link CompressionType#ZSTD} Compress with Zstd codec. Since Pulsar 2.3, Zstd can only be used
```

This should be a famous abbr.